src/test/java/app/coronawarn/verification/VerificationApplicationExternalTest.java

```diff
@@ -293,11 +375,10 @@ public void callGetRegistrationTokenByTeleTan() throws Exception {
     given(this.tanService.isTeleTanValid(TestUtils.TEST_TELE_TAN)).willReturn(true);
     given(this.tanService.getEntityByTan(TestUtils.TEST_TELE_TAN)).willReturn(Optional.of(TestUtils.getTeleTanTestData()));
 
-    mockMvc.perform(post(TestUtils.PREFIX_API_VERSION + TestUtils.REGISTRATION_TOKEN_URI)
+    assertTrue(mockMvc.perform(post(TestUtils.PREFIX_API_VERSION + TestUtils.REGISTRATION_TOKEN_URI)
       .contentType(MediaType.APPLICATION_JSON)
       .content(TestUtils.getAsJsonFormat(request)))
-      .andExpect(status().isCreated())
-      .andExpect(jsonPath("$.registrationToken").exists());
+      .andReturn().getAsyncResult().toString().contains("201"));
```

Comment:
see my comment regarding ``` asyncDispatch() ``` - I think this would be a better solution for this part as well.